### PR TITLE
feat(daemon): add Claude Code agent support

### PR DESCRIPTION
**Pull request by casey-brooks**
## Summary
- add Claude settings writer/base URL helper and tests
- add Claude daemon wiring/handler plus dispatch updates
- add claude-sdk-go dependency and handler tests

## Testing
- GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go vet ./...
- GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go build ./...
- GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go test ./...

#70

### CODE REVIEW

**Comment by casey-brooks** [comment]
## Test & Lint Summary
- Lint: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go vet ./... (no issues)
- Build: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go build ./... (success)
- Tests: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go test ./... (passed: 7 packages; failed: 0; skipped: 1 no test files)

File: go.mod [hunk 1]
```diff
@@ -1,11 +1,10 @@
 module github.com/agynio/agynd-cli
 
-go 1.24.0
-
-toolchain go1.24.13
+go 1.26.1
```
Comment (noa-lucent) [inline go.mod]: [minor] The Go version was bumped from `1.24.0` (with `toolchain go1.24.13`) to `go 1.26.1` and the `toolchain` directive was dropped. This is unrelated to Claude Code support and isn't mentioned in the PR description. If `claude-sdk-go` requires Go 1.26+, please note that in the PR body. Otherwise, keep the version bump in a separate commit/PR so that this change is traceable.

**Review by noa-lucent — DISMISSED** [review]
Clean, well-structured implementation. The Claude daemon follows the established patterns (AGN, Codex) consistently — settings writer, constructor, message handler, and tests are all in line with the codebase conventions.

One minor note on the go.mod version bump (1.24 → 1.26.1) — please document the reason in the PR description or split it out if unrelated to this change.

All issue requirements from #70 are addressed. Tests are thorough and cover the right edge cases.

**Comment by casey-brooks** [comment]
## Test & Lint Summary
- Lint: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go vet ./... (no issues)
- Build: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go build ./... (success)
- Tests: GOPRIVATE=github.com/agynio/* GONOSUMDB=github.com/agynio/* GONOPROXY=github.com/agynio/* go test ./... (passed: 7 packages; failed: 0; skipped: 1 no test files)
